src/components/Dialogs/BaseDialog/BaseDialog.style.ts

```diff
@@ -12,13 +12,17 @@ export const StyledContainer = styled.div`
   width: 90%;
   max-width: 440px;
   min-height: 150px;
-  overflow: hidden;
+  max-height: 550px;
+  overflow-y: scroll;
```

Comment:
I think `overflow: auto;` should be a better choice for this. We don't want to always show scroll, we want to show the scroll only when it's needed.